service/history/historyBuilder.go

```diff
@@ -30,8 +30,9 @@ import (
 )
 
 const (
-	firstEventID int64 = 1
-	emptyEventID int64 = -23
+	firstEventID    int64 = 1
+	emptyEventID    int64 = -23
+	bufferedEventID int64 = -123
```

Comment:
Just curious why choose number -123? (also why emptyEventID -23)

Reply:
I just pick a random number that is unique and is less than 0.